lib/axlsx/workbook/worksheet/cell.rb

```diff
@@ -376,6 +377,8 @@ def is_formula?
     end
 
     def is_array_formula?
+      return false if escape_formulas
+
       type == :string && @value.to_s.start_with?('{=') && @value.to_s.end_with?('}')
```

Comment:
```suggestion
      type == :string && @value.to_s.start_with?('{=') && @value.to_s.end_with?('}')
```

I'm not confident about this check, but again, I'm no expert of how excel works

I may logically think that this should also check for `{@`, `{+`, etc. but actually I wasn't able to replicate an issue with a formual starting with `@` or `+`.

I was thinking to something like ARRAY_FORMULA_PREFIXES which takes FORMULA_PREFIXES and maps it with a leading `{`. If you have a reproducible test case with a formula starting with any other character than `=` it could help me

Reply:
We should verify that `{@`, `{|` etc. are actually an issue...

Reply:
I can't even replicate issues with formulas starting with `+` on my excel version